src/show/pages/public/Register/index.jsx

```diff
@@ -0,0 +1,95 @@
+import { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+
+import Paths from 'process/routes/paths';
+import { register } from 'process/slices/userSlice';
+
+import { Button, TextField } from 'show/components';
+
+const CreateAccount = () => {
+  const [client, setClient] = useState({
+    email: '',
+    password: '',
+    first_name: '',
+    last_name: ''
+  });
+
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+
+  const registerUser = e => {
+    e.preventDefault();
+    dispatch(
+      register({
+        email: client.email,
+        password: client.password,
+        first_name: client.first_name,
+        last_name: client.last_name
+      }),
```

Comment:
Client is already an object containing all of those you need, don't bother doing them like this when you can send the whole object.